src/components/Input/Input.types.ts

```diff
@@ -6,10 +6,12 @@
  */
 import { IUseFieldProps } from '../../hooks';
 
+type DefaultHtmlInputValueType = React.InputHTMLAttributes<HTMLInputElement>['value'];
```

Comment:
I like this pattern a lot. Let the displaying component decide the TDisplayValue through its prop interface.